Dockerfile

```diff
@@ -35,6 +36,7 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-reco
         media-types \
         procps \
         nano \
+        curl \
```

Comment:
Do we need this?

Reply:
We could need it for healthcheck? 
If we want to have a docker-compose file withfull configuration of OZ and ON we need to check if django apps are running. 
For example here https://github.com/open-zaak/open-zaak/pull/1576/files 

Perhaps there is another solution, but I'm not aware of it